src/XamlIl.Cecil/CecilMethod.cs

```diff
@@ -110,6 +136,11 @@ public UnresolvedMethod(string name)
             
             public bool Equals(IXamlIlMethod other) => other == this;
 
+            public IXamlIlMethod MakeGenericMethod(IReadOnlyList<IXamlIlType> typeArguments)
```

Comment:
`UnresolvedMethod` class seems to be never used, so it can be safely deleted instead.